docker-compose.yml

```
@@ -23,18 +23,17 @@ services:
-      MYSQL_PASSWORD: ${MYSQL_PASSWORD}
+      MYSQL_USER: merico
+      MYSQL_PASSWORD: merico
       TZ: UTC
-      ENCRYPTION_SECRET: ${ENCRYPTION_SECRET}
     command: --character-set-server=utf8mb4
       --collation-server=utf8mb4_bin
       --skip-log-bin
 
   grafana:
-    image: devlake.docker.scarf.sh/apache/devlake-dashboard:v1.0.1   
+    image: devlake.docker.scarf.sh/apache/devlake-dashboard:v1.0.0   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)